README.md

```diff
@@ -1,8 +1,35 @@
 # local-storage-operator
 Operator for local storage
 
+```mermaid
+graph LR
+    PVC -->|Requests local storage using a StorageClass| StorageClass
+    StorageClass -->|Defines a LocalVolume| LocalVolume
+    LocalVolume -->|Represents a physical storage device already mounted on a node| Node
+    Pod -->|References a PVC| PVC
+    PV -->|References| LocalVolume
+    StorageClass -->|Creates| PV
+    VolumeAttachment -->|Correlates| Node
+    VolumeAttachment -->|Correlates| PV
+    LSO((Local Storage Operator)) -->|Manages| LocalVolume
+    LSO((Local Storage Operator)) -->|Manages| StorageClass
+    LocalVolumeSet -->|Discovers local storage devices on node| Node
+    LocalVolumeSet -->|Automatically manages LocalVolume objects|LocalVolume
+```
+
 ## Deploying with OLM
 Instructions to deploy on OCP >= 4.2 using OLM can be found [here](docs/deploy-with-olm.md)
 
 ## Using the must-gather image with the local storage operator
 Instructions for using the local storage's must-gather image can be found [here](docs/must-gather.md)
+
+## How does a user interact with LSO?
+```mermaid
+graph LR
+    User((User)) -->|Enables| LocalVolumeDiscovery
```

Comment:
IMO, user `Creates` LocalVolumeDiscovery.
And LSO creates / manages LocalVolumeDiscoveryResult as the consequence.

Reply:
Agreed. I made this change and added LVDR as well in https://github.com/openshift/local-storage-operator/commit/f8acb21ec74b51b9fdfde6ab9f8c0a8d71ad3ea9#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R25